template/express-rest-api/Dockerfile.dev-deps

```diff
@@ -2,10 +2,12 @@
 
 FROM --platform=${BUILDPLATFORM:-<%- platformName %>} node:20-alpine AS dev-deps
 
-WORKDIR /workdir
+RUN corepack enable pnpm
+RUN pnpm config set store-dir /root/.pnpm-store
 
-COPY package.json yarn.lock ./
+WORKDIR /workdir
 
-RUN \
-  --mount=type=secret,id=npm,dst=/root/.npmrc \
-  yarn install --frozen-lockfile --ignore-optional --non-interactive
+RUN --mount=type=bind,source=package.json,target=package.json \
```

Comment:
We can skip binding on package.json if we're using fetch yeah?

Reply:
Ah, and I need to include `.npmrc` instead 👍